libsolidity/ast/Types.cpp

```diff
@@ -1478,6 +1478,15 @@ TypePointer StringLiteralType::mobileType() const
 	return TypeProvider::stringMemory();
 }
 
+bool StringLiteralType::isValidASCII() const
```

Comment:
This helper is not needed since it is implemented in SyntaxChecker, I'd even argue the `isValidUTF8` helper is useless below and the single location it is used should just use `validateUTF8`.